terraform/keycloak-sandbox/templates/keycloak.json.tpl

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "name": "keycloak",
+    "image": "${app_image}",
+    "cpu": 0,
+    "secrets": [
+      {
+        "valueFrom": "${url_path}",
+        "name": "DB_ADDR"
+      },
+      {
+        "valueFrom": "${username_path}",
+        "name": "DB_USER"
+      },
+      {
+        "valueFrom": "${password_path}",
+        "name": "DB_PASSWORD"
+      },
+      {
+        "valueFrom": "${admin_user_path}",
+        "name": "KEYCLOAK_USER"
+      },
+      {
+        "valueFrom": "${admin_password_path}",
+        "name": "KEYCLOAK_PASSWORD"
+      },
+      {
+        "valueFrom" : "${client_secret_path}",
+        "name": "CLIENT_SECRET"
+      },
+      {
+        "valueFrom": "${backend_checks_client_secret_path}",
+        "name": "BACKEND_CHECKS_CLIENT_SECRET"
+      },
+      {
+        "valueFrom": "${realm_admin_client_secret_path}",
+        "name": "REALM_ADMIN_CLIENT_SECRET"
+      },
+      {
+        "valueFrom": "${configuration_properties_path}",
+        "name": "KEYCLOAK_CONFIGURATION_PROPERTIES"
+      },
+      {
+        "valueFrom": "${user_admin_client_secret_path}",
+        "name": "USER_ADMIN_CLIENT_SECRET"
+      }
+    ],
+    "environment": [
+      {
+        "name" : "FRONTEND_URL",
+        "value" : "${frontend_url}"
+      },
+      {
+        "name" : "DB_VENDOR",
+        "value" : "postgres"
+      },
+      {
+        "name" : "KEYCLOAK_IMPORT",
+        "value": "/tmp/tdr-realm.json"
+      }
+    ],
+    "networkMode": "awsvpc",
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "/ecs/keycloak-${app_environment}",
+        "awslogs-region": "${aws_region}",
+        "awslogs-stream-prefix": "ecs"
+      }
+    },
+    "portMappings": [
+      {
+        "containerPort": 8080,
```

Comment:
Identation

Reply:
Fixed, thanks.

That reminds me - I'll create a separate PR to add a Jenkins build to run `terraform fmt` and git-secrets.

Reply:
Added in PR #3